app/[lang]/chapters/[slug]/layout.tsx

```diff
@@ -1,13 +1,21 @@
-import Navbar from 'components/Navbar'
+'use client'
+
+import Navbar, { NavbarMobile } from 'components/Navbar'
 import { chapters } from 'content'
+import useMediaQuery from 'hooks/useMediaQuery'
 
-export default async function Layout({ children, params }) {
+export default function Layout({ children, params }) {
   const isValidChapter = chapters[params.slug]
+  const isSmallScreen = useMediaQuery('(max-width: 767px)')
 
   return (
     <div className="ch1-background flex flex-col">
       <div className="fix-grow-issue flex min-h-screen flex-col blur">
-        {isValidChapter && <Navbar params={params} />}
+        {isValidChapter && isSmallScreen ? (
+          <NavbarMobile params={params} />
+        ) : (
+          <Navbar params={params} />
```

Comment:
the `Navbar` component should not be displayed to the user if the chapter is not valid. for example this page - https://saving-satoshi-git-fork-shaavan-navbar-update-2-savingsatoshi.vercel.app/en/chapters/chapter2d should only display the text `Coming Soon...`. https://savingsatoshi.com/en/chapters/chapter2d (what it looks like on saving satoshi)

Reply:
Good catch @vinayaksh42! I think we should also make sure the height of the content doesn't exceed that of the viewport on that page @shaavan.

Reply:
Thanks for catching that, @vinayaksh42.

I have corrected the code to display the navbar only when the chapter is valid.

Addressed in d884f7a5c4caca0536fa42a21125dbe7739111fe

Reply:
@afterburn
I have also addressed your suggestion in d884f7a5c4caca0536fa42a21125dbe7739111fe

The container div on the "Coming Soon" page doesn't overflow from the viewport and triggers the scroll bar.

However, it is better to split this commit into a separate PR since this PR focuses on the Navbar update.